src/MiNET/MiNET/Player.cs

```diff
@@ -3627,7 +3627,7 @@ protected virtual void OnTicked(PlayerEventArgs e)
 		public virtual void HandleMcpeNetworkStackLatencyPacket(McpeNetworkStackLatencyPacket message)
```

Comment:
This handler need to check the boolean to see if it is a request or a response. Otherwise the server will loop it around. Thanks for catching that @dktapps

Reply:
Also don't reply to client requested pings. So can just as well ignore this one entirely.